src/utils/helpers.ts

```diff
@@ -0,0 +1,19 @@
+export const getIdFromURL = (url: string): number => {
+  const id = url.slice(0, -1).split('/').pop();
+
+  if (typeof id === 'string') {
+    return parseInt(id);
+  }
+
+  return 0;
+};
+
+export const isTypeOf = <T extends Record<string, unknown>>(value: unknown, key: keyof T): value is T => {
+  return value instanceof Object && key in value;
```

Comment:
`key in value` potential dangerous condition.
If value = null it will throw an exception.